src/components/InputWithLabel/InputWithLabel.js

```diff
@@ -29,4 +30,12 @@ const InputWithLabel = ({
   );
 };
 
+InputWithLabel.propTypes = {
+  id: PropTypes.string,
+  name: PropTypes.string,
+  type: PropTypes.string,
+  value: PropTypes.string,
+  onChange: PropTypes.func,
+};
+
```

Comment:
You should use the .isRequired modifier (e.g.: onChange: PropTypes.func.isRequired) in most of the propTypes (in all the components you use propTypes), in this particular case the type prop could be omitted because you're sending a default value as the actual props (type="text").